### DISCUSSION TITLE
How do I get the last action back?

### DISCUSSION THREAD

**Discussion by Lexus2016** [LiveDuo/destack#119] [General]
There is a situation when after placing Meraki UI - Navigation (Last block) there is no button to delete or move it. I add other components - there is a button on all of them, but not on this one. I had to completely reset the project and start all over again. How can you solve such problems? Where is the file with the configuration of what has already been done?

![Image 2024-05-16 20-58-56](https://github.com/LiveDuo/destack/assets/25110505/46f94972-4fd7-4417-a818-7fb7c7173a62)

**Comment by LiveDuo** [comment]
@Lexus2016 Thanks for reporting.

It's not exactly a configuration but an HTML file and it's located in [data/default.html](https://github.com/LiveDuo/destack-starter/blob/main/data/default.html).

If you can reproduce the problem maybe in the form of a screen recording I could have a look further into the bug.
